src/components/dropdownSearch/DropdownSearch.tsx

```diff
@@ -127,7 +130,7 @@ export class DropdownSearch extends React.Component<IDropdownSearchProps, void>
           : value;
 
         return (
-          <li key={option.value}
+          <li key={UUID.generate()}
```

Comment:
you cant do that sry, all the optimization for the 5000 elements is broken with this generated